src/client/CodeEditor/json1PresenceDisplay.ts

```diff
@@ -88,14 +88,15 @@ export const json1PresenceDisplay = ({
               presence,
             );
 
-            // Generate user color based on the id
-  const userColor = new ColorHash({ lightness: 0.75 }).rgb(id).join(',');
+// Initialize or update presence with user color
+const initializePresence = (id: string, presence: Presence): Presence => {
+  return assignUserColor(presence, id);
```

Comment:
This may not work, the presence object may be re-initialized or something, as this is managed by ShareDB. Better not try to modify it.